docs/src/components/DocsPageBuilder/ComponentScreenshots.scss

```diff
@@ -31,6 +32,7 @@
 
     @include bpk-breakpoint-above-mobile {
       width: calc(50% - #{$bpk-spacing-base});
+      min-width: calc(50% - #{$bpk-spacing-base});
```

Comment:
So this makes them larger if the viewport is wide enough, right?

Reply:
Yeah they should be exactly the same as before, but now when you have more than two it won't make them tiny, it'll wrap them onto a new row